packages/concerto-core/changelog.txt

```diff
@@ -24,7 +24,7 @@
 # Note that the latest public API is documented using JSDocs and is available in api.txt.
 #
 
-Version 0.82.6 {03fa6481ffdf0e58cb110c6f24009d18} 2020-02-26
```

Comment:
Bump version in package.json to align with this.

Reply:
hm. always an issue there since we don't have that version number yet... I probably should open an issue about that.

Reply:
Not completely fixed, since technically this should go onto version `0.82.9` but that version number is created through the release scripts.

Reply:
Fixed the version checker for a little leeway...